@xen-orchestra/web/src/components/pool/system/PoolManagement.vue

```diff
@@ -5,44 +5,47 @@
     </UiTitle>
     <VtsStateHero v-if="!areHostsReady" format="card" busy size="medium" />
     <template v-else>
-      <VtsQuickInfoRow :label="t('master')">
+      <UiLabelValue :label="t('master')">
         <template #value>
-          <UiLink v-if="primaryHost" icon="fa:server" :to="`/host/${pool.master}/`" size="medium">
-            {{ primaryHost.name_label }}
+          <UiLink v-if="primaryHost" icon="fa:server" :to="`/host/${pool.master}/`" size="medium" class="link">
```

Comment:
```suggestion
          <UiLink v-if="primaryHost" icon="fa:server" :to="`/host/${pool.master}/`" size="medium">
```
I don't think this class is necessary